includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -2456,6 +2456,48 @@ public function schedule_order_tracking( $order_id, $order = null ) {
 		}
 	}
 
+	/**
+	 * Create a payment intent without confirming the intent.
+	 *
+	 * @param WC_Order $order - Order based on which to create intent.
+	 * @param array    $payment_methods - A list of allowed payment methods. Eg. card, card_present.
+	 * @param ?string  $capture_method - Controls when the funds will be captured from the customer's account ("automatic" or "manual").
```

Comment:
`@param ?string` perhaps?

Reply:
Isn't it the same already?

Reply:
Nope, `?string` in PHP types is `string|null`. And we are requiring string only.

Reply:
Gotcha! Fixed here a313299 ✅